### ISSUE TITLE
Ironic does not support ilo5

### ISSUE THREAD

**Issue by mel1nn** [metal3-io/baremetal-operator#775]
I want to provision a baremetal server (HPE) using the ilo5 bmc.
When I apply the related baremetalhost on my cluster, an error occurs in the Ironic pod:
` No valid host was found. Reason: No conductor service registered which supports driver ilo5 for conductor group`.

Thanks for your help.

**Comment by elfosardo** [comment]
hi @mel1nn , ironic does support ilo5 and baremetal operator has added also support for that type of hardware some months ago.
Looking at the error you're seeing I suspect the ilo5 hardware type is not included in the ironic configuration.
You can see here https://docs.openstack.org/ironic/latest/admin/drivers/ilo.html how to configure it.

**Comment by mel1nn** [comment]
Thanks for your answer. But why this configuration is not included by default in the ironic image (https://github.com/metal3-io/ironic-image) ?

**Comment by hardys** [comment]
> Thanks for your answer. But why this configuration is not included by default in the ironic image (https://github.com/metal3-io/ironic-image) ?

Agreed - it looks like @ankit-kumar5 added the initial ilo support via https://github.com/metal3-io/baremetal-operator/pull/595 and there are also some related virtualmedia changes from @honza and @dhellmann looking at the git history.

I suspect not updating the ironic-image default configuration was an oversight, and we should add the ilo drivers to those enabled in https://github.com/metal3-io/ironic-image/blob/master/ironic.conf.j2 @elfosardo would you agree?

**Comment by elfosardo** [comment]
@hardys I do agree, ilo5 should be there

**Comment by maelk** [comment]
This should probably be transferred to the ironic-image repo. Shall I do it ?

**Comment by mel1nn** [comment]
Yes it should ! Thanks
